_unused_topics/arch-platform-operators.adoc

```diff
@@ -22,4 +22,4 @@ Operator Lifecycle Manager (OLM) introduces a new type of Operator called _platf
 
 Using the existing cluster capabilities feature in {product-title}, cluster administrators can already disable a subset of Cluster Version Operator-based (CVO) components considered non-essential to the initial payload prior to cluster installation. Platform Operators iterate on this model by providing additional customization options. Through the platform Operator mechanism, which relies on resources from the RukPak component, OLM-based Operators can now be installed at cluster installation time and can block cluster rollout if the Operator fails to install successfully.
 
-In {product-title} 4.12, this Technology Preview release focuses on the basic platform Operator mechanism and builds a foundation for expanding the concept in upcoming releases. You can use the cluster-wide `PlatformOperator` API to configure Operators before or after cluster creation on clusters that have enabled the `TechPreviewNoUpgrade` feature set.
\ No newline at end of file
+In {product-title} 4.12, this Technology Preview release focuses on the basic platform Operator mechanism and builds a foundation for expanding the concept in upcoming releases. You can use the cluster-wide `PlatformOperator` API to configure Operators before or after cluster creation on clusters that have enabled the `TechPreviewNoUpgrade` feature set.
```

Comment:
I guess the `PlatformOperator` has already been removed.

Reply:
Correct. This was a module I missed in my previous PR. 

I have updated the PR so that the module is deleted: https://github.com/openshift/openshift-docs/pull/76370/files#diff-1919c988286f5598e6e3870c1aeb505e6399b138b68d61809789963f24fda404